src/OrchardCore/OrchardCore.ContentManagement/ContentDefinitionManager.cs

```diff
@@ -72,11 +72,6 @@ public async Task<ContentTypeDefinition> LoadTypeDefinitionAsync(string name)
         {
             ArgumentException.ThrowIfNullOrEmpty(name);
 
-            if (_scopedTypeDefinitions.TryGetValue(name, out var typeDefinition))
-            {
-                return typeDefinition;
-            }
```

Comment:
@sebastienros should be the problem here, the Loadxx family of methods should no longer fetch content from the cache
/cc @MikeAlhayek

Reply:
Hi @MikeAlhayek , do I need to revert these changes? Or do I keep them?
Because there is already a cache in the `ContentDefinitionStore ` and a second-level cache would make the logic more complex

Reply:
If we use both `MemoryCache` and Redis caching, you may have problems enabling distributed caching and multi-server deployments